src/ontology/mondo-edit.obo

```diff
@@ -61765,21 +61763,20 @@ intersection_of: disease_has_location UBERON:0002137 ! aortic valve
 
 [Term]
 id: MONDO:0003804
-name: blood protein disease
+name: obsolete blood protein disease
 comment: Reason: out of scope specific to autosomal disorder. Term to consider: none.
-subset: obsoletion_candidate
 synonym: "blood Protein disorder" RELATED [MESH:D001796]
 synonym: "blood protein disorder" EXACT [DOID:620]
 synonym: "disorder, blood Protein" RELATED [MESH:D001796]
 synonym: "disorders, blood Protein" RELATED [MESH:D001796]
 synonym: "Protein disorder, blood" RELATED [MESH:D001796]
 synonym: "Protein disorders, blood" RELATED [MESH:D001796]
-xref: DOID:620 {source="MONDO:equivalentTo"}
-xref: MESH:D001796 {source="MONDO:equivalentTo", source="DOID:620"}
-xref: UMLS:C0005830 {source="MONDO:equivalentTo", source="DOID:620"}
-is_a: MONDO:0005570 {source="DOID:620", source="MESH:D001796"} ! hematologic disorder
+xref: DOID:620 {source="MONDO:obsoleteEquivalent"}
+xref: MESH:D001796 {source="MONDO:obsoleteEquivalent", source="DOID:620"}
+xref: UMLS:C0005830 {source="MONDO:obsoleteEquivalent", source="DOID:620"}
 relationship: seeAlso https://github.com/monarch-initiative/mondo/issues/4624
-property_value: IAO:0006012 "2022-06-01" xsd:string
+property_value: IAO:0000231 "out of scope" xsd:string
```

Comment:
@nicolevasilevsky : I didn't add a source annotation that comes from this [list of exclusion reasons](https://mondo.readthedocs.io/en/latest/editors-guide/exclusion-reasons/). I didn't know which one was appropriate.

Reply:
see comment on ticket

Reply:
@nicolevasilevsky 
I updated the comment. I still don't know whether there is an appropriate source annotation though.